distribution/lib/Standard/Database/0.0.0-dev/src/DB_Column.enso

```diff
@@ -1505,9 +1505,17 @@ type DB_Column
              import Standard.Examples
 
              example_contains = Examples.text_column_1.regex_match (regex "A.*M")
-    regex_match self (pattern : Column | Text | Any) -> Column =
-        _ = [pattern]
-        Error.throw (Unsupported_Database_Operation.Error ("regex_match"))
+    regex_match self (pattern : DB_Column | Text | Any) -> DB_Column =
```

Comment:
Is `Any` actually valid here?

- DB_Column and Text are
- Regex should be
- Any seems invalid?